Exercise-4/flattenArray.js

```diff
@@ -0,0 +1,39 @@
+// Function that checks the level of nesting of an array and flattens the array into a single-dimension array
+
+const nestedArray1 = [1, 2, [3, 4, [5, 5]]];
+
+const nestedArray2 = [1, 2, [3, 4, [5, 6, [7, 8, [9, 10, [11, 12]]]]]];
+
+const maxDepth = (inputArray, depth = 1) => {
+  // Initialize a variable to track the maximum depth encountered
```

Comment:
the name of the variable should be enough to explain what it will hold or close to it.